app/js/views/vendorreceipt.js

```diff
@@ -70,9 +69,11 @@ define(
             // },
             {
                 name: "supplier",
-                label: "Supplier",
+                label: "Vendor",
                     control: ModelSelectControl,
-                    room_model_module: SupplierChoice
+                    room_model_module: SupplierChoice,
+                    // helpMessage found -> old forms.py line 257
+                    // helpMessage: "or add a new vendor using the form on the right"
```

Comment:
I created a new issue to track this
https://github.com/babybunny/rebuildingtogethercaptain/issues/346

so you can remove the comment.  there is another similar case in stafftime.js.